packages/common/src/components/Button/elements.ts

```diff
@@ -1,164 +1,177 @@
-};
-
-const getColor = ({
-  disabled,
-  secondary,
-  theme: internalTheme,
-}: OptionProps) => {
-  if (disabled) return theme.background2.lighten(1.5)();
-  if (secondary)
-    return internalTheme.light
-      ? 'rgba(0, 0, 0, 0.75)'
-      : 'rgba(255, 255, 255, 0.75)';
-
-  return 'white';
+  if (secondary) {
+    return css`
+      background-color: #66b9f4;
+    `;
+  }
+  if (red) {
+    return css`
+      background-color: #f27777;
+    `;
+  }
+  if (theme[`button.hoverBackground`]) {
+    return css`
+      background-color: ${theme[`button.hoverBackground`]};
+    `;
+  }
+  return css`
+    background-color: #66b9f4;
+  `;
 };
 
-const getHoverColor = ({ secondary, disabled }: OptionProps) => {
-  if (disabled) return '';
-  if (secondary) return 'color: white';
-
-  return '';
+const getColor = ({ disabled, secondary, theme }: OptionProps) => {
+  if (disabled) {
+    return theme.background2.lighten(1.5)();
+  }
+  if (secondary) {
+    return theme.light ? `rgba(0, 0, 0, 0.75)` : `rgba(255, 255, 255, 0.75)`;
+  }
+  return `white`;
 };
 
 const getBorder = ({
-  theme: internalTheme,
+  theme,
   secondary,
   danger,
   red,
   disabled,
 }: OptionProps) => {
-  if (disabled)
-    return internalTheme.light
-      ? '2px solid rgba(0, 0, 0, 0.3)'
-      : '2px solid #161A1C';
-  if (secondary) return `2px solid #66B9F4`;
-  if (red) return '2px solid #F27777';
-  if (danger) return '2px solid #E25D6A';
-  if (internalTheme && internalTheme['button.hoverBackground']) {
-    return `2px solid ${internalTheme['button.hoverBackground']}`;
+  if (disabled) {
+    return theme.light ? `2px solid rgba(0, 0, 0, 0.3)` : `2px solid #161A1C`;
   }
-
-  return '2px solid #66B9F4';
+  if (secondary) {
+    return `2px solid #66B9F4`;
+  }
+  if (red) {
+    return `2px solid #F27777`;
+  }
+  if (danger) {
+    return `2px solid #E25D6A`;
+  }
+  if (theme && theme[`button.hoverBackground`]) {
+    return `2px solid ${theme[`button.hoverBackground`]}`;
+  }
+  return `2px solid #66B9F4`;
 };
 
-const styles = css<{
-  disabled?: boolean;
-  secondary?: boolean;
-  danger?: boolean;
-  red?: boolean;
+export interface IBaseProps extends OptionProps {
   block?: boolean;
   small?: boolean;
-}>`
-  transition: 0.3s ease all;
-  font-family: Poppins, Roboto, sans-serif;
-
-  border: none;
-  outline: none;
-  ${props => getBackgroundColor(props)};
-  background-size: 720%;
-
-  border: ${props => getBorder(props)};
+}
+
+export const buttonStyles = ({
+  disabled = false,
+  secondary = false,
+  block = false,
+  small = false,
+  theme,
+}) => css`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: ${block ? '100%' : 'inherit'};
+  padding: ${small ? `2px 6px` : `0.65rem 2.25rem`};
+  border: ${getBorder};
   border-radius: 4px;
-
+  ${getBackgroundColor};
   box-sizing: border-box;
-  font-size: 1.125em;
+  color: ${getColor};
+  ${small ? theme.fonts.secondary.normal : theme.fonts.secondary.medium};
```

Comment:
The buttons styles have changed with this update and some paddings are uneven now (especially in notifications). Let's change the button styles back to how they were before before we're going to implement the design system.